python/pysmurf/client/command/smurf_command.py

```diff
@@ -4545,36 +4614,13 @@ def set_hemt_enable(self, disable=False, **kwargs):
                 self.rtm_spi_max_root + self._hemt_v_enable_reg,
                 2, **kwargs)
 
-    def set_hemt_gate_voltage(self, voltage, override=False,
-                              **kwargs):
-        """
-        Sets the HEMT gate voltage in units of volts.
-
-        Args
-        ----
-        voltage : float
-            The voltage applied to the HEMT gate. Must be between 0
-            and .75.
-        override bool, optional, default False
-            Override thee limits on HEMT gate voltage.
-        """
-        self.set_hemt_enable()
-        if (voltage > self._hemt_gate_max_voltage or voltage <
-                self._hemt_gate_min_voltage ) and not override:
-            self.log(
-                'Input voltage too high. Not doing anything.' +
-                ' If you really want it higher, use the ' +
-                'override optional arg.')
-        else:
-            self.set_hemt_bias(int(voltage/self._hemt_bit_to_V),
-                override=override, **kwargs)
-
     _hemt_v_reg = 'HemtBiasDacDataRegCh[33]'
 
     def set_hemt_bias(self, val, override=False, **kwargs):
         """
-        Sets the HEMT voltage in units of bits. Need to figure out the
-        conversion into real units.
+        Sets the HEMT voltage in units of bits. This applies
+        to both HEMT1 and HEMT2 because there is only one register
+        to set the bias with, HemtBiasDacDataRegCh[33].
```

Comment:
These statements are a bit confusing. Does this mean that the set_hemt_bias function will set the bias for both Hemt1 and Hemt2 to the same value?

Reply:
Yeah exactly. There are actually 32 DACs which we can easily interact with using set_rtm_slow_dac_volt, but for some reason there is also address 33 which I don't fully understand, which is why I added that blurb in. I think I should rewrite set_hemt_bias to set set_rtm_slow_dac_volt on the HEMT1 DAC address, and add set_hemt2_bias for the HEMT2 DAC. That should work, unless I'm missing something.

Reply:
I'll save those changes for another PR, I should just modify the 50k stuff first